<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #152028;
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>610</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1F2933;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="VaultButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>210</y>
      <width>91</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/vault.png</normaloff>:/Images/SideBar/vault.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>350</y>
      <width>51</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1F2933;
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/settings.png</normaloff>:/Images/SideBar/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ExportButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1F2933;
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../CredentialMenu/files.qrc">
      <normaloff>:/Images/SideBar/export.png</normaloff>:/Images/SideBar/export.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-10</y>
     <width>921</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Password Generator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CopyButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../CredentialMenu/files.qrc">
     <normaloff>:/Images/CredentialMenu/copy.png</normaloff>
     <normalon>:/Images/copy.png</normalon>:/Images/CredentialMenu/copy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="CenterFrame">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="GeneratedPasswordLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 2px solid white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Color: white;
border: 2px solid white;</string>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="CopyPasswordLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Color: white;</string>
    </property>
    <property name="text">
     <string>Copy Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="GeneratorTweaksFrame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="NumberCharactersCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Color: white;</string>
    </property>
    <property name="text">
     <string>Change Total Number Of Characters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TotalCharactersLine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IncludeButton">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;
color: white;</string>
    </property>
    <property name="text">
     <string>Add Character To Include</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExcludeButton">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>100</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;
color: white;</string>
    </property>
    <property name="text">
     <string>Add Character To Exclude</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="IncludeExcludeFrame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="AddCharactersLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: none;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Use Fields To Add Characters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IE_FIELD_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCharactersButton">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>170</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;
color: white;</string>
    </property>
    <property name="text">
     <string>Include/Exclude Characters</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="AlphaNumericBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Color: white;</string>
   </property>
   <property name="text">
    <string>Use Only Letters and Numbers</string>
   </property>
  </widget>
  <zorder>GeneratorTweaksFrame</zorder>
  <zorder>CenterFrame</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>frame</zorder>
  <zorder>CopyButton</zorder>
  <zorder>AlphaNumericBox</zorder>
  <zorder>IncludeExcludeFrame</zorder>
 </widget>
 <resources>
  <include location="../CredentialMenu/files.qrc"/>
 </resources>
 <connections/>
</ui>
